src/CanSatKitBMP280.h

```diff
@@ -5,83 +5,132 @@
+		 */
+		BMP280();
 
-		char begin();
-		char begin(int sdaPin, int sclPin);
-			// call pressure.begin() to initialize BMP280 before use
-			// returns 1 if success, 0 if failure (i2C connection problem.)
-				
-		short getOversampling();
-		char  setOversampling(short oss);
-		
-		char startMeasurment();
-			// command BMP280 to start a pressure measurement
-			// oversampling: 0 - 3 for oversampling value
-			// returns (number of ms to wait) for success, 0 for fail
+		/**
+		 * @brief Initialize sensor and calibrate.
+		 * 
+		 * @return `true` if initialization suceeded, `false` otherwise (eg. i2C connection problem)
+		 */
+		bool begin();
 		
-		char calcTemperature(double &T, double &uT);
-			// calculation the true temperature from the given uncalibrated Temperature 
-			
-		char calcPressure(double &P, double uP);
-			//calculation for measuring pressure.
-
-		char getError();
-			// If any library command fails, you can retrieve an extended
-			// error code using this command. Errors are from the wire library: 
-			// 0 = Success
-			// 1 = Data too long to fit in transmit buffer
-			// 2 = Received NACK on transmit of address
-			// 3 = Received NACK on transmit of data
-			// 4 = Other error
-			
-		char readTemperatureAndPressure(double& T, double& P);
+		/**
+		 * @brief Set oversampling ratio.
+		 * Should be one of the following: 0, 1, 2, 3, 4, 16.
+		 * 
+		 * \rst
+		 * +---------------+-------------+
+         * | Oversampling  | Delay (ms)  |
+         * +===============+=============+
+         * | 0             | 8           |
+         * +---------------+-------------+
+		 * | 1             | 10          |
+         * +---------------+-------------+
+		 * | 2             | 15          |
+         * +---------------+-------------+
+		 * | 3             | 24          |
+         * +---------------+-------------+
+		 * | 4             | 45          |
+         * +---------------+-------------+
+		 * | 16            | 80          |
+         * +---------------+-------------+
+		 * 
+		 * \endrst
+		 * 
+		 * Higher oversampling means longer measurement, but more accurate result
+		 * 
+		 * @param oversampling radio
+		 */
+		void setOversampling(uint8_t oversampling);
+
+		/**
+		 * @brief Get previously set oversampling.
+		 * 
+		 * @return uint8_t oversampling set
+		 */
+		uint8_t getOversampling();
+
+		/**
+		 * @brief Perform temperature and pressure measurement.
+		 * This is blocking function (return always new value).
+		 * For non-blocking usage, see startMeasurment() and readTemperatureAndPressure()
+		 * 
+		 * @param T temperature read from the device
+		 * @param P pressure read from the device
+		 * @return bool Measurement status
+		 */
+		bool measureTemperatureAndPressure(double& T, double& P);
+
+
+		/**
+		 * @brief Begin a measurement cycle.
+		 * This function returns the delay before result will be available to read.
+		 * 
+		 * @return unsigned int delay to read 
```

Comment:
consider `unsigned int delay to read measurement result (ms)`

Reply:
done